integration-tests/oidc-wiremock/src/test/java/io/quarkus/it/keycloak/KeycloakTestResource.java

```diff
@@ -22,24 +21,23 @@
     @Override
     public Map<String, String> start() {
 
-        server = new WireMockServer(options().port(8180));
+        server = new WireMockServer();
         server.start();
 
-        WireMock.configureFor(8180);
         WireMock.stubFor(
                 get(urlEqualTo("/auth/realms/quarkus/.well-known/openid-configuration"))
                         .willReturn(aResponse()
                                 .withHeader("Content-Type", MediaType.APPLICATION_JSON)
                                 .withBody("{\n" +
-                                        "    \"authorization_endpoint\": \"http://localhost:8180/authenticate\",\n" +
-                                        "    \"end_session_endpoint\": \"http://localhost:8180/logout\",\n" +
+                                        "    \"authorization_endpoint\": \"http://localhost:8080/authenticate\",\n" +
```

Comment:
Another option would be to use `server.getOptions().portNumber()`

Reply:
@gastaldi @machi1990 it also has `server.baseUrl()` so I've used it and also removed the redundant discovery values which are not needed for the mock testing; it should be very easy for the users to mock the bearer token authentication; code flow would have something more similar to the original, CC @cemnura